modin/pandas/test/test_series.py

```diff
@@ -3388,54 +3388,46 @@ def test_update(data, other_data):
     df_equals(modin_series, pandas_series)
 
 
+@pytest.mark.parametrize("sort", [True, False])
```

Comment:
There is `bool_arg_values` which also includes `None` in the list of values to test that it doesn't cause problems.
What about other value_counts parameters, are they supported?

Reply:
added all of the `value_counts` parameters to the test